airframe-sql/src/main/scala/wvlet/airframe/sql/parser/SQLGenerator.scala

```diff
@@ -379,15 +379,15 @@ object SQLGenerator extends LogSupport {
         s"(${printExpression(expr)})"
       case a: Alias =>
         val e = printExpression(a.expr)
-        s"${e} AS ${a.name}"
+        s"${e} AS ${printNameWithQuotationsIfNeeded(a.name)}"
```

Comment:
Unfortunately, using `a.sqlExpr` does not work as it's broken (it generates invalid SQL). It would've been nice since, currently, the generation logic is awkwardly happening here and within each expression. 

However, the changes would be way more massive.